### DISCUSSION TITLE
Please add support for multiple accounts.

### DISCUSSION THREAD

**Discussion by eblot** [community/community#3265] [Mobile] [answered]
Having to chose either my professional or private account is definitely not great.

Thanks

**Comment by jsoref** [comment]
The are many discussions with this request. Please note that on average if you're asking, you're probably in violation of the terms of service. 
https://github.com/community/community/discussions/39693#discussioncomment-5754343

**Comment by eliperkins** [comment, marked as answer]
Hey @eblot and folks, we've shipped functionality this week to allow you to log in with multiple accounts, like your personal and work accounts, at a the same time! [Read more about it in our Changelog post](https://github.blog/changelog/2023-08-15-log-in-with-multiple-github-accounts-on-github-mobile/).

<img width="1200" alt="image" src="https://github.com/community/community/assets/1051453/d2ba902e-2199-4625-948a-532d64a6f44e">

Download or update GitHub Mobile today from the [Apple App Store](https://apps.apple.com/us/app/github/id1477376905) or [Google Play Store](https://play.google.com/store/apps/details?id=com.github.android) to get started.
